src/mgmt/deploy-director-vm.sh

```diff
@@ -357,8 +343,8 @@ virt-install --name director \
   --vcpus 8 \
   --hvm \
   --os-type linux \
-  --os-variant rhel7 \
-  --disk /store/data/images/director.img,bus=virtio,size=80 \
+  --os-variant rhel8.1 \
```

Comment:
Do we have to spell out the sub version, e.g, rhel8.1 vs rhel8?  Seems like we should stick with just rhel8.

Reply:
Just verified that in rhel8 you have to be specific with the os name and it does not accept just any name. Used "osinfo-query os | grep rhel" command to list down available os names